vault/resource_mount_test.go

```diff
@@ -217,6 +217,56 @@ func TestResourceMount_ExternalEntropyAccess(t *testing.T) {
 	})
 }
 
+func TestResourceMountMangedKeys(t *testing.T) {
```

Comment:
Since it's quite a different config and setup, and since the test cases are also slightly tailored, I figured it might be worth it to have a separate test for it; the other mount tests use config functions and checks which are shared between multiple tests, and I didn't want to break any of the existing code with my work or trample over it. Thoughts?